src/Components/prueba.js

```
@@ -0,0 +1,14 @@
+function BADfunction() {
+    var a = 10
+    var b = "20"
+    if(a = b){
+        alert("a is equal to b")
+    }
+    for(i=0;i<10;i++){
+        setTimeout(function(){
+            console.log(i)
+        },1000)
+    }
+    eval("console.log('This is unsafe!')")
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)